model/src/main/java/org/openremote/model/manager/Credentials.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Can be replaced with `org.openremote.model.auth.UsernamePassword`, feel free to adjust the ts code to use this type also